content/blog/2020-11-25-metadata/contents.lr

```diff
@@ -0,0 +1,47 @@
+title: Metadata: Schema Reference Document and Future Work
+---
+author: Sara Gonzales
+---
+body:
+
+On 17 and 24 November 2020 the InvenioRDM project partners met in two telecons to discuss the [Metadata Reference](https://inveniordm.docs.cern.ch/reference/metadata/) document.
+
+
+![](metadata_reference.png)
+
+
+These conversations were a follow-up to review of the document and collection of questions in a [Google Doc](https://docs.google.com/document/d/1djBrPMaeVH4IlpFADMJSpgd5JJ1xIiPY5GwLs87Wwjw/edit?usp=sharing). The conversations helped to confirm many answers to questions about the data model that had been posed in the Doc, such as: whether ‘type’ is required for Creators (no - only for Contributors), are Resource Types customizable (yes), and which file is shown in the previewer when a record has multiple files (the first, unless otherwise specified). See the Doc for responses to additional questions. 
```

Comment:
```suggestion
These conversations were a follow-up to the community's review of the document. Feedback and questions were collected in a [Google Doc](https://docs.google.com/document/d/1djBrPMaeVH4IlpFADMJSpgd5JJ1xIiPY5GwLs87Wwjw/edit?usp=sharing) prior to the conversations. The conversations helped to confirm many answers to questions about the data model that had been posed in the Doc, such as: whether ‘role’ is required for Creators (no - only for Contributors), are Resource Types customizable (yes), and which file is shown in the previewer when a record has multiple files (the first, unless otherwise specified). See the Doc for responses to additional questions. 
```